res/skins/LateNight/skin.xml

```diff
@@ -46,12 +44,13 @@
       <attribute config_key="[Microphone],show_microphone">0</attribute>
       <attribute config_key="[Vinylcontrol],show_vinylcontrol">0</attribute>
       <attribute config_key="[PreviewDeck],show_previewdeck">0</attribute>
-      <attribute config_key="[Master],mixer_4decks_status">1</attribute>
       <attribute config_key="[Master],show_mixer">1</attribute>
       <attribute config_key="[Master],show_mics">0</attribute>
       <attribute config_key="[Library],show_coverart">1</attribute>
       <attribute config_key="[Library],show_library">1</attribute>
       <attribute config_key="[Library],show_current">1</attribute>
+      <attribute config_key="[Master],show_4decks">0</attribute>
```

Comment:
This conflicts with the persist attribute set below. 
You cannot set a start up value for the skin at on hand and state that it should persist. 
What is the worth of a persist flag if you override the value just after the start? 

I would prefer to have the setup parameter in one place at the very start of the skin only and the connection parameter in the rest of the string.

Here: 
Value is only set when not persist or no saved value available.
If you need to force th co to a value set remove the persist attribute. 
```
<attribute config_key="[Master],show_4decks" persist="true">0</attribute>
``` 

Elsewere: 
No "persist" allowed. This was hard to maintain anyway, badly placed and a source of bug reports.
```
<Connection>
    <ConfigKey>[Master],show_4decks</ConfigKey>
    <ButtonState>LeftButton</ButtonState>
</Connection>

```